Easydict/App/Localizable.xcstrings

```diff
@@ -6359,7 +6359,7 @@
         "sk" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Secret Access Key"
```

Comment:
This doesn't need to be translated.

Reply:
Do I have to do something now? Or is it OK now?

Reply:
@Svec-Tomas Please check the latest review, thanks.